### ISSUE TITLE
Downloads just stopping regardless of which site

### ISSUE THREAD

**Issue by karehaqt** [riderkick/FMD#927]
For some reason my downloads will just stop part way through so I'm having to constantly babysit them.  Any ideas as to why, I've messed around with the Connection tab settings but nothing seems to work, they'll merrily download then just stop dead at 0 Bps.

**Comment by Tmp341** [comment]
Have you tried fresh install? Backup your "data", "works" and if you have, just "config\accounts.db" file. Delete everything and extract fmd again and restore your backed up files.

**Comment by karehaqt** [comment]
@Tmp341 I'll give it a try after work tonight and see how I get on, I might also try some older versions as this has just been a recent thing.

**Comment by karehaqt** [comment]
This is my log as everything stopped again just incase anyone can spot anything [Log](https://i.imgur.com/tDRVH18.png)

**Comment by Tmp341** [comment]
This one happened on MangaDex it seems. Have you tried other ones?

**Comment by karehaqt** [comment]
Not yet as most of my bookmarks are for Mangadex since I moved my Batato favourites to them.

**Comment by Tmp341** [comment]
Try one of them on Kissmanga or another one.

**Comment by riderkick** [comment]
Lower the timeout and increase the retry times.

**Comment by karehaqt** [comment]
Thanks for the tip riderkick, will give that a try to after work :)

**Comment by karehaqt** [comment]
Tried all suggestions, everything bar Mangadex works fine, Mangadex just grinds to a halt, wonder if that's by design on their part.

**Comment by leoteesh** [comment]
mangago stopped working again. it just stops downloading by itself and if it does finish, it fails

![image](https://user-images.githubusercontent.com/21075197/36072803-55414e22-0f1e-11e8-9eb0-2b7e07803ebb.png)

**Comment by Tmp341** [comment]
MangaGo has an eye on FMD. When it is solved, they change something again and again.

**Comment by leoteesh** [comment]
yeah, still getting errors and fmd terminating even after the update :( so sad
![image](https://user-images.githubusercontent.com/21075197/36118927-80cf1628-1036-11e8-9c40-d35217aab766.png)

**Comment by leoteesh** [comment]
mango was working this morning but now it stopped again :( - such a shame

![image](https://user-images.githubusercontent.com/21075197/36174088-146ba49c-1103-11e8-8204-f2a09cd8482d.png)

**Comment by leoteesh** [comment]
does anyone know if downloads with mangago have been fixed with the recent updates? 
many thanks

**Comment by Tmp341** [comment]
Nope. They are fighting like they are in a war with fmd.

**Comment by leoteesh** [comment]
It's such a shame because they seem to be the only one who has more mangas and updates on their site
